model_analyzer/config/run/run_config_generator.py

```diff
@@ -135,7 +137,35 @@ def generate_run_config_for_model_sweep(self, model, model_sweep):
                     RunConfig(model.model_name(), model_config, perf_config,
                               model.triton_server_environment()))
 
+    def _apply_model_config_defaults(self, config_dict, cpu_only):
+        if "instance_group" not in config_dict:
+            config_dict["instance_group"] = [{}]
+        for instance_group in config_dict["instance_group"]:
+            self._apply_instance_group_defaults(instance_group, cpu_only)
+
+    def _apply_instance_group_defaults(self, instance_group, cpu_only):
```

Comment:
We should run the default config as is without any modifications as it might change behavior.

Reply:
reverted